cirrus/server/cirrus/main.py

```diff
@@ -109,6 +110,39 @@ def initialize_glean():
     return pings, metrics
 
 
+def get_metrics_data(
+    enrolled_partial_configuration: Dict[str, Any]
+) -> List[tuple[str, str, str]]:
```

Comment:
I'm not a super big fan of anonymous tuples. I'd like to see either a dataclass or `typing.NamedTuple`:

```python
class Metrics(typing.NamedTuple):
    experiment_slug: str
    branch_slug: str
    experiment_tupe: str
```